offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -221,7 +221,8 @@ class UpdatePersonView extends CreatePersonView{
      * @return boolean which indicates if a person update can be triggered
      */
     protected boolean allValuesValid() {
-        return createPersonViewModel.firstNameValid \
+        return createPersonViewModel.academicTitleValid \
```

Comment:
If this is required then I would expect the view showing the required indicator or the Combobox being filled from the get-go. Otherwise, the user ends up with a view where the create/update button is not clickable even though every field with a required indicator was filled.

Reply:
![image](https://user-images.githubusercontent.com/11536497/120338570-a4f10600-c2f4-11eb-9353-99f58a8a4588.png)

Reply:
Good point 👍